README.md

```diff
@@ -320,6 +321,31 @@ end
 
 ```
 
+### Container Analysis (Alpha)
+
+- [google-cloud-container_analysis README](google-cloud-container_analysis/README.md)
+- [google-cloud-container_analysis API documentation](https://googleapis.github.io/google-cloud-ruby/docs/google-cloud-container_analysis/latest)
+- [google-cloud-container_analysis on RubyGems](https://rubygems.org/gems/google-cloud-container_analysis)
+- [Container Analysis documentation](https://cloud.google.com/container-registry/docs/container-analysis/)
+
+#### Quick Start
+
+```sh
+$ gem install google-cloud-container_analysis
+```
+
+#### Preview
```

Comment:
This example is not present in [google-cloud-container_analysis/README.md)](https://github.com/googleapis/google-cloud-ruby/blob/master/google-cloud-container_analysis/README.md). Do you want to add it there as well, or omit it here?

Reply:
Added it to the container_analysis readme. (This required me to remove the readme from synth, but meh.)